doc/manual/manual.tex

```diff
@@ -7945,17 +7945,17 @@ \subsubsection{Continental extension}
 \begin{figure}
 \centering
 \includegraphics[width=\textwidth]{cookbooks/continental_extension/continental_extension_cookbook_strainrate_5e6yr.png}
-\caption{\it Strain rate ($s^{-1}$) after 5e6 years of extension. The black line marks the 500 $^{o}C$ isotherm.} 
+\caption{\it Strain rate ($s^{-1}$) after 5e6 years of extension. The black line marks the 550 $^{o}C$ isotherm.} 
```

Comment:
While you're there, can you change this to `$550^\circ$C`? That creates the proper "degree" symbol. Same for the other figures below.